src/Web/Slack/AesonUtils.hs

```diff
@@ -108,6 +110,15 @@ snakeCaseOptions =
     , constructorTagModifier = camelTo2 '_'
     }
 
+-- | 'snakeCaseOptions' that eats trailing underscores. This is so that you can
+-- have a field called "type_".
+snakeCaseOptionsEatTrailing :: Options
```

Comment:
The name is really long, but maybe it should be longer? `snakeCaseOptionsEatTrailingUnderscores`